mobile_app/make_android_build.py

```diff
@@ -0,0 +1,29 @@
+"""
+Invoke this script to kick off a new android build
+"""
+
+import logging
+import sys
+
+from mobile_app.make_build import make_build
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+def make_android_build():
+    """
+    Kicks off a new android app build, asking the user a few
+    additional questions
+    """
+    make_build(
+        "git@github.com:edx/edx-app-android.git",
```

Comment:
I think that's a reasonable requirement. You need write access to the build repo to make this work anyway.